data/names/ru.json

```diff
@@ -5870,7 +5870,624 @@
+      [ "Лэйн", "Lane" ],
+      [ "Лэнд", "Land" ],
+      [ "Ландерс", "Landers" ],
+      [ "Лемонс", "Lemons" ],
+      [ "Фокс", "Fox" ],
+      [ "Литлджон", "Littlejohn" ],
+      [ "Лоуренс", "Lawrence" ],
+      [ "Медоуз", "Meadows" ],
+      [ "Луи", "Louis" ],
+      [ "Лу", "Luu" ],
+      [ "Боуман", "Bowman" ],
+      [ "Бэст", "Best" ],
+      [ "Лэйси", "Lacey" ],
+      [ "Лэндри", "Landry" ],
+      [ "Лэрд", "Laird" ],
+      [ "Лэтэм", "Latham" ],
+      [ "Люнг", "Leung" ],
+      [ "Ма", "Ma" ],
+      [ "Майрик", "Myrick" ],
+      [ "Макафи", "McAfee" ],
+      [ "Макговерн", "McGovern" ],
+      [ "Макдональд", "McDonald" ],
+      [ "Макдоноу", "McDonough" ],
+      [ "Макдэниел", "McDaniel" ],
+      [ "Макинтайр", "McIntyre" ],
+      [ "Маккерди", "McCurdy" ],
+      [ "Макколлум", "McCollum" ],
+      [ "Маккрари", "McCrary" ],
+      [ "Макмэн", "McMahan" ],
+      [ "Макмэн", "McMahon" ],
+      [ "Макмиллан", "McMillan" ],
+      [ "Макналти", "McNulty" ],
+      [ "Макнамара", "McNamara" ],
+      [ "Макнейр", "McNair" ],
+      [ "Макнелли", "McNally" ],
+      [ "Макнили", "McNeely" ],
+      [ "Макфадден", "McFadden" ],
+      [ "Макфарланд", "McFarland" ],
+      [ "Макферсон", "McPherson" ],
+      [ "Макхью", "McHugh" ],
+      [ "Литтл", "Little" ],
+      [ "Маллинз", "Mullins" ],
+      [ "Маррокен", "Marroquin" ],
+      [ "Мартинс", "Martins" ],
+      [ "Мастерс", "Masters" ],
+      [ "Медейрос", "Medeiros" ],
+      [ "Медли", "Medley" ],
+      [ "Меза", "Meza" ],
+      [ "Мейерс", "Meyers" ],
+      [ "Майо", "Mayo" ],
+      [ "Мэйсон", "Mason" ],
+      [ "Мелансон", "Melanson" ],
+      [ "Мело", "Melo" ],
+      [ "Мелтон", "Melton" ],
+      [ "Мендес", "Mendes" ],
+      [ "Меррилл", "Merrill" ],
+      [ "Мёрфи", "Murphy" ],
+      [ "Маркс", "Marks" ],
+      [ "Мизе", "Mize" ],
+      [ "Мирелес", "Mireles" ],
+      [ "Мозес", "Moses" ],
+      [ "Мойя", "Moya" ],
+      [ "Мок", "Mock" ],
+      [ "Янг", "Young" ],
+      [ "Хаммер", "Hammer" ],
+      [ "Монк", "Monk" ],
+      [ "Мониз", "Moniz" ],
+      [ "Монтанес", "Montanez" ],
+      [ "Монтейру", "Monteiro" ],
+      [ "Морен", "Morin" ],
+      [ "Мориарти", "Moriarty" ],
+      [ "Фрост", "Frost" ],
+      [ "Морс", "Morse" ],
+      [ "Симен", "Seaman" ],
+      [ "Моултон", "Moulton" ],
+      [ "Мосс", "Moss" ],
+      [ "Мэйберри", "Mayberry" ],
+      [ "Мэйн", "Main" ],
+      [ "Мейес", "Mayes" ],
+      [ "Мэллори", "Mallory" ],
+      [ "Мэсси", "Massey" ],
+      [ "Мэттсон", "Mattson" ],
+      [ "Надь", "Nagy" ],
+      [ "Нахера", "Najera" ],
+      [ "Надо", "Nadeau" ],
+      [ "Ын", "Ng" ],
+      [ "Невилл", "Neville" ],
+      [ "Уикс", "Weeks" ],
+      [ "Нойманн", "Neumann" ],
+      [ "Барроуз", "Burrows" ],
+      [ "Ньюджент", "Nugent" ],
+      [ "Нэйлор", "Naylor" ],
+      [ "Нейпир", "Napier" ],
+      [ "О'Брайен", "O'Brien" ],
+      [ "О'Коннор", "O'Connor" ],
+      [ "Обриен", "Obrien" ],
```

Comment:
I suspect this is meant to be a different spelling of O'Brien, as much as some of the following ones being an apostrophy-less spelling of their respective Irish counterparts. The reasons for the lack of apostrophe are unclear to me, as Google tries to pivot *hard* towards O'Brien even when I specify the "reduced" variant. (The lack of results for it as a separate surname also suggests this is just a different spelling.)

All of which is a long-winded way for me to say: it would sound like the original, i.e. `Обрайен`.